reference/configuration/framework.rst

```diff
@@ -2847,32 +2847,6 @@ annotation changes). For performance reasons, it is recommended to disable
 debug mode in production, which will happen automatically if you use the
 default value.
 
-secrets
```

Comment:
Duplication due to merge conflicts

https://github.com/symfony/symfony-docs/pull/19537#issuecomment-1935726505